.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish (pre)release to npm
+
+on:
+  release:
+    # include release and pre-release, exclude release draft
+    types: [released,prereleased]
```

Comment:
I think it would be better to just subscribe to the published type, rather than these two, as noted here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

The release and prereleased types will not trigger if we draft a release and then later publish it.

Reply:
Ah I misread this part for `published`

> "or draft of a release was published"

It sounded to me that saving a draft would trigger the action, which wasn't meant like that. Thank you, will fix.